mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -0,0 +1,445 @@
+                                       endDate=endDateClimo,
+                                       calendar=calendar)
+    print '\n  List of files for climatologies:\n{} through {}'.format(
+           inputFilesClimo[0], inputFilesClimo[-1])
+    startYearClimo = config.getint('climatology', 'startYear')
+    endYearClimo = config.getint('climatology', 'endYear')
+
+    plotsDirectory = buildConfigFullPath(config, 'output', 'plotsSubdirectory')
+    mainRunName = config.get('runs', 'mainRunName')
+    #   Then a list necessary for the streamfunctionMOC Atlantic timeseries
+    startDateTseries = config.get('timeSeries', 'startDate')
+    endDateTseries = config.get('timeSeries', 'endDate')
+    inputFilesTseries = streams.readpath(streamName,
+                                         startDate=startDateTseries,
+                                         endDate=endDateTseries,
+                                         calendar=calendar)
+    print '  List of files for timeSeries:\n{} through {}'.format(
+           inputFilesTseries[0], inputFilesTseries[-1])
+    startYearTseries = config.getint('timeSeries', 'startYear')
+    endYearTseries = config.getint('timeSeries', 'endYear')
+
+    try:
+        restartFile = streams.readpath('restart')[0]
+    except ValueError:
+        raise IOError('No MPAS-O restart file found: need at least one '
+                      'restart file for MOC calculation')
+
+    sectionNameClimo = 'streamfunctionMOC'
+    sectionNameTseries = 'timeSeriesMOC'
+
+    # Plotting variables
+    colormapName = plt.get_cmap(config.get(sectionNameClimo, 'colormapName'))
+    colormapIndices = config.getExpression(sectionNameClimo, 'colormapIndices')
+    colormapName = cols.ListedColormap(colormapName(colormapIndices),
+                                       'colormapName')
+    colorbarLabel = '[Sv]'
+    movingAveragePoints = config.getint(sectionNameTseries,
+                                        'movingAveragePoints')
+
+    # Check whether MOC Analysis Member is enabled
+    if mocAnalysisMemberFlag == '.true.':
+
+        # Add a moc_analisysMember_processing
+        print '*** MOC Analysis Member is on ***'
+
+    else:
+
+        # Load mesh related variables
+        ncFile = netCDF4.Dataset(restartFile, mode='r')
+        dvEdge = ncFile.variables['dvEdge'][:]
+        areaCell = ncFile.variables['areaCell'][:]
+        refBottomDepth = ncFile.variables['refBottomDepth'][:]
+        latCell = ncFile.variables['latCell'][:]
+        latCell = latCell * 180.0 / np.pi  # convert to degree
+        ncFile.close()
+
+        nVertLevels = len(refBottomDepth)
+        refTopDepth = np.zeros(nVertLevels+1)
+        refTopDepth[1:nVertLevels+1] = refBottomDepth[0:nVertLevels]
+        refLayerThickness = np.zeros(nVertLevels)
+        refLayerThickness[0] = refBottomDepth[0]
+        refLayerThickness[1:nVertLevels] =  refBottomDepth[1:nVertLevels] - \
```

Comment:
This was one specific case when flake8 didn't like what I did, no matter what. which is why I decided to choose the best indent for readability. 
I think this was the only option that flake8 liked, from what I remember:
```
refLayerThickness[1:nVertLevels] =  refBottomDepth[1:nVertLevels] - \
    refBottomDepth[0:nVertLevels-1]
```